### DISCUSSION TITLE
Lockstep versions are annoying

### DISCUSSION THREAD

**Discussion by Faithfinder** [TanStack/query#6374] [General] [answered]
I find myself annoyed at common release notes feed for all the query packages. I think most developers only care about one of them at a time. If I review release notes it's probably to decide whether is safe to upgrade, and I want the react package, and notes like these are a lot of noise to read through:

![image](https://github.com/TanStack/query/assets/25051063/baa5a733-afa6-4734-b72e-55a06e683cd7)

**Comment by TkDodo** [comment, marked as answer]
you can search for `react` to filter it down:

https://github.com/TanStack/query/releases?q=react&expanded=true

**Reply by Faithfinder** [reply]
I never noticed the search bar there, lol.

I guess the query would need to be `react OR core` to get everything that can affect you.

**Reply by TkDodo** [reply]
When core is released, react is released too

**Reply by Faithfinder** [reply]
Is it always?
![image](https://github.com/TanStack/query/assets/25051063/b3381d6f-d6b4-4d1d-bc87-b4793955cf50)

**Reply by TkDodo** [reply]
yes, it is. 5.8.2 was a broken release. The release notes were made manually and I didn't list the packages, and something wasn't right:
- https://github.com/TanStack/query/discussions/6365#discussioncomment-7551340

react-query re-exports everything from the core so that you don't have to build a dependency on the core:

https://github.com/TanStack/query/blob/b20f1f79b89e8f019a19e80aad579e14f28d2f4f/packages/react-query/src/index.ts#L3-L4

so if something in the core changes, everything that depends on it will also get released.
